.github/ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "🚀 Feature request"
+about: I have a suggestion
+---
+
+### Your checklist for this issue
+
+🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.
```

Comment:
I might have some ideas.

Perhaps we could ask a few questions:

- [ ] Link to any upstream changes that might be required (for example Jenkins Core pull request)
- [ ] Have you considered an extension plugin
- [ ] ?

Perhaps we could link to a list of possible extension they can provide to JCasC such as secret source.

@timja @ewelinawilkosz @ndeloof perhaps you have some more suggestions

Reply:
I'd like to keep it short, so what really matters is the description of the feature request. "Link to any upstream changes" is a good idea too. In many cases I can imagine answering "Have you considered an extension plugin" may be tricky since feature requests might be coming from non-developers. We can leave it but I don't feel we NEED it. Not a blocker though. You can decide @casz and merge :)